pkg/server/util/regentryutil/fetch.go

```diff
@@ -3,16 +3,24 @@ package regentryutil
 import (
 	"context"
 	"errors"
+	"time"
 
 	"github.com/golang/protobuf/ptypes/wrappers"
 	"github.com/spiffe/spire/pkg/common/util"
 	"github.com/spiffe/spire/pkg/server/plugin/datastore"
 	"github.com/spiffe/spire/proto/spire/common"
 )
 
+const cacheFetchEntriesTTL = 5 * time.Second
```

Comment:
Are the same results achieved if we use a 1 second TTL instead of 5? I think it'd be beneficial to try using a lower TTL to reduce overall propagation delay in the system, especially if 5 seconds isn't strictly necessary in order to get the benefits.

Reply:
yeah I didn't notice significant improvement with TTL of 5 when compared with 1 sec. I think we can start off with 1 sec.